scripts/tests/core-integration.test.mjs

```diff
@@ -28,67 +28,62 @@ describe("CORE Integration Tests", () => {
-
-      // Simulate the filtering logic from findSourceDocFiles
-      const filtered = files.filter(f => f.endsWith(".md")).filter(f => !(f === "README.md"));
+      const allFiles = await glob(`${docsDir}/**/*.md`);
+      const relativeFiles = allFiles.map(f => path.relative(docsDir, f));
+      const filtered = relativeFiles
+        .filter(f => f.endsWith(".md"))
+        .filter(f => !(f === "README.md"));
 
       expect(filtered).not.toContain("README.md");
       expect(filtered).toContain("010_user-guide/README.md");
       expect(filtered).toContain("getting-started.md");
     });
   });
 
-  describe("Root Community Files", () => {
+  describe("Root Contributing and Community Files Routing", () => {
     it.each([
-      ["SECURITY.md", "# Security Policy\n\nPlease report vulnerabilities...", "Security Policy"],
-      ["CODE-OF-CONDUCT.md", "# Code of Conduct\n\nBe respectful...", "Code of Conduct"],
-      ["SUPPORT.md", "# Support\n\nGet help with Pepr...", "Support"],
-    ])("should process %s from repository root", async (filename, content, expectedText) => {
-      const filePath = path.join(coreRepo, filename);
-
-      await fs.writeFile(filePath, content, "utf8");
-
-      const exists = await fs
-        .access(filePath)
-        .then(() => true)
-        .catch(() => false);
-      expect(exists).toBe(true);
-
-      const fileContent = await fs.readFile(filePath, "utf8");
-      expect(fileContent).toContain(expectedText);
+      ["SECURITY.md", "community/security.md"],
+      ["CODE-OF-CONDUCT.md", "contribute/code-of-conduct.md"],
+      ["SUPPORT.md", "community/support.md"],
+    ])(
+      "should route %s to correct destination directory",
+      async (sourceFile, expectedDestination) => {
+        const ROOT_MD_MAPPINGS = [
+          { sources: ["SECURITY.md"], target: "community/security.md" },
+          { sources: ["CODE-OF-CONDUCT.md"], target: "contribute/code-of-conduct.md" },
+          { sources: ["SUPPORT.md"], target: "community/support.md" },
+        ];
+
+        const mapping = ROOT_MD_MAPPINGS.find(m => m.sources.includes(sourceFile));
+
+        expect(mapping).toBeDefined();
+        expect(mapping.target).toBe(expectedDestination);
+      },
+    );
+
+    it("should place SECURITY.md in community/ not contribute/", async () => {
+      const ROOT_MD_MAPPINGS = [
+        { sources: ["SECURITY.md"], target: "community/security.md" },
+        { sources: ["SUPPORT.md"], target: "community/support.md" },
+        { sources: ["CODE-OF-CONDUCT.md"], target: "contribute/code-of-conduct.md" },
+      ];
+
+      const securityMapping = ROOT_MD_MAPPINGS.find(m => m.sources.includes("SECURITY.md"));
+      const conductMapping = ROOT_MD_MAPPINGS.find(m => m.sources.includes("CODE-OF-CONDUCT.md"));
+
+      expect(securityMapping.target).toContain("community/");
+      expect(conductMapping.target).toContain("contribute/");
+      expect(securityMapping.target).not.toContain("contribute/");
```

Comment:
What's the behavior under test for this one? I see that we create a list of expected inputs and outputs. Then we seem to ensure that the outputs that we've defined match the outputs that we've defined? 

This feels like something along the lines of testing getters & setters, which is _sometimes_ worth doing. Do you think this is one of those cases? Here's [a blog post](https://medium.com/@mabogert64/the-argument-to-test-your-getters-and-setters-b97a62c285e7) that might be a handy reference.